scripts/rollup.config.js

```diff
@@ -76,7 +76,8 @@ const options = {
         compress: {passes: 2}
       }),
       strip: strip({
-        functions: ['TEST_ONLY_*']
+        functions: ['TEST_ONLY_*'],
+        debugger: false
```

Comment:
can we add `sourceMap: false` while we're at it?
```suggestion
        debugger: false,
        sourceMap: false
```